examples/http-swagger/src/example/server.clj

```diff
@@ -52,7 +52,7 @@
        ["/openapi.json"
         {:get {:no-doc true
                :openapi {:info {:title "my-api"
-                                :description "with reitit-http"
+                                :description "openap-docs with reitit-http"
```

Comment:
I know this is merged, but there's a small typo here:

Should be `openapi-docs ...`

Reply:
in #592